targets/ramips-mt7620

```diff
@@ -7,3 +7,5 @@ factory
 
 device gl-mt750 gl-mt750
 factory
+
+device nexx-wt3020-8m wt3020-8M
```

Comment:
I wonder if it's worth it to create aliases for the different 8M models (WT3020AD, WT3020F, WT3020H).